### PR TITLE
[DNM] Columba-class Mining Carrier

### PR DESCRIPTION
**Pull request by goober3**
Adds a new subshuttle-oriented ship for covert and dangerous mining operations enacted via a small mining shuttle. Features a full telecomms setup and ports Fikou's spyglass rework for an EXOCOM-themed miner monitoring kit.

## SCREEEEEEEEEEEEEEEENSHOTS

thething
^this ship now uses outdated ion thrusters, from #1585 

subshuttle

awagga

image

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game

Funny mining niche with subshutles for a faction that doesn't have any mining ships.

## Changelog

:cl:
add: The Columba-class Mining Carrier, a Nanotrasen mining ship with a small subshuttle.
tweak: Spyglasses are now accessed via an action rather than a verb
tweak: Removed bandanas from mixed walldrobes to prevent locker overfill
/:cl:

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/wardrobe.dm [hunk 1]
```diff
@@ -189,12 +189,6 @@
 	new /obj/item/clothing/under/color/jumpskirt/lightpurple(src)
 	new /obj/item/clothing/under/color/green(src)
 	new /obj/item/clothing/under/color/jumpskirt/green(src)
-	new /obj/item/clothing/mask/bandana/red(src)
-	new /obj/item/clothing/mask/bandana/red(src)
-	new /obj/item/clothing/mask/bandana/blue(src)
-	new /obj/item/clothing/mask/bandana/blue(src)
-	new /obj/item/clothing/mask/bandana/gold(src)
-	new /obj/item/clothing/mask/bandana/gold(src)
```
Comment (Erikafox) [inline code/game/objects/structures/crates_lockers/closets/wardrobe.dm:197]: what's up with this?

File: _maps/shuttles/subshuttles/Subshuttle Catalog.txt [hunk 2]
```diff
@@ -2,3 +2,9 @@ Name = "Example Name Subshuttle"
 Size = "1x3"
 Purpose = "Showing people how to fill this document in"
 File Path = "_maps\shuttles\subshuttles\example.dmm"
+
+
+Name = "Mining Shuttle"
```
Comment (Erikafox) [inline _maps/shuttles/subshuttles/Subshuttle Catalog.txt]: You can give it a bit more of a name

File: _maps/shuttles/shiptest/nanotrasen_columba.dmm [hunk 3]
```diff
@@ -0,0 +1,2528 @@
...
+/area/ship/engineering)
+"Uu" = (
+/obj/machinery/door/airlock/external,
+/obj/docking_port/mobile{
+	callTime = 250;
+	dir = 2;
+	launch_status = 0;
+	name = "Mining Carrier";
+	port_direction = 8;
+	preferred_direction = 4
```
Comment (Erikafox) [inline _maps/shuttles/shiptest/nanotrasen_columba.dmm]: set the can_move_docking_ports var to true, or 1. Don't remember which it takes, but it needs it.

**Review by Erikafox — COMMENTED** [review]
StrongDMM_0v4QGRD0VL
Cafe!
image
I'd move the APC down a tile here, swap it with the CO2 poster, and maybe put a pump against the wall to express that that is where the ship's waste is going.
image
If you're gonna have the hub on the same vertical segment as the bridge, I would have the rail follow it.
StrongDMM_VLi430TIlA
This vent won't look nice when someone cracks the tile open to look at the terminal, I'd move it back a tile.
StrongDMM_UyMe8KvsSc
Generate Icon States gives you your choice of the kirbyplants. 
StrongDMM_UVcdk3CvT3
StrongDMM_j737JW0YVS
What's up here?
StrongDMM_2K1dHrRBtH
Cut the shelter capsules and give them marker beacons
StrongDMM_3Ns4xvXLEU
bad vent dir
Your decals are really faded because you're using the transparent variants, wanna make sure that's intentional
StrongDMM_1TBMUM80s1
doubled up decal

I would hit up another maptainer about Nanotrasen stylings, because I vaguely recall something about them using titanium, but I also recall a lot of other things that turn out to be wrong.
Also I don't see the subshuttle in the pr.

**Comment by Erikafox** [comment]
paintdotnet_TrHgCONoSN
Almost forgot. I think the crew quarters could stand to be a bit bigger. I used my artistic talents to create a sort of mock up of how that might happen.

**Comment by Apogee-dev** [comment]
Remember that there are Nanotrasen-specific outfits already defined for several jobs, including captain and miner. The default outfit datum for a job is often independent-styled. If you aren’t sure how to specify outfit datums, check out the config files for other ships. Most ships use custom outfits at this point.

**Comment by goober3** [comment]
> Remember that there are Nanotrasen-specific outfits already defined for several jobs, including captain and miner. The default outfit datum for a job is often independent-styled. If you aren’t sure how to specify outfit datums, check out the config files for other ships. Most ships use custom outfits at this point.

already fixed this! just couldn't commit due to bad internet

**Comment by goober3** [comment]
> paintdotnet_TrHgCONoSN Almost forgot. I think the crew quarters could stand to be a bit bigger. I used my artistic talents to create a sort of mock up of how that might happen.

i'm calling the director of the louvre...

File: code/game/objects/structures/crates_lockers/closets/wardrobe.dm (hunk 1 above)
Comment (goober3) [reply code/game/objects/structures/crates_lockers/closets/wardrobe.dm:197]: mixed walldrobes (wosets?) were spawning with too much THINGS in them, and would not close with everything in them. so I removed all of the bandanas, there's probably a better solution to this.

**Comment by Erikafox** [comment]
firefox_QFGuOuekYq
the front of the ship isn't linked to the back
firefox_eR0OdvE4nK
needs a smidge more variety. Try some carpets, or splines.

**Comment by goober3** [comment]
image
image
greebled...

**Comment by Zevotech** [comment]
finally. a ship i could see myself buying and enjoying regularly.

**Comment by Erikafox** [comment]
image
I'm still not happy with this, but I don't really know how to fix it beyond start again with a new layout for it. I'd hit up another maptainer, because someone else probably has a better idea of how to do it.

Engi looks better though

**Comment by Zevotech** [comment]
> I'm still not happy with this, but I don't really know how to fix it beyond start again with a new layout for it. I'd hit up another > maptainer, because someone else probably has a better idea of how to do it.
> 
> Engi looks better though

bar stools perhaps?

**Comment by goober3** [comment]
> image I'm still not happy with this, but I don't really know how to fix it beyond start again with a new layout for it. I'd hit up another maptainer, because someone else probably has a better idea of how to do it.
> 
> Engi looks better though

ok... how does THIS look, "professional" "maptainer"?

image

**Comment by goober3** [comment]
image
reworked canteen again. hopefully in a way that appeases the maptainers. also look at that OUTPOST console. that's an OUTPOST console.

**Comment by goober3** [comment]
also had to add a 1 SHIP LIMIT. to fix telecomms relays linking with each other and making problems. if anyone has a better way of fixing that, please tell!

**Comment by goober3** [comment]
this Ruin has been UPDATED.

**Comment by thgvr** [comment]
Quite a nice ship, from subshuttle testing. Excited to have it around after subshuttles are finished.

**Comment by goober3** [comment]
> Quite a nice ship, from subshuttle testing. Excited to have it around after subshuttles are finished.

thank u! I am actually planning on remaking this ship from scratch. once subshuttles are merged. in a more or less scrunkled way idk.

**Comment by goober3** [comment]
i'm KILLING this PR

**Pull request closed without merging**